MYOB.API.SDK/SDK/Contracts/Version2/Contact/Customer.cs

```diff
@@ -1,4 +1,4 @@
-using MYOB.AccountRight.SDK.Contracts.Version2.GeneralLedger;
+﻿using MYOB.AccountRight.SDK.Contracts.Version2.GeneralLedger;
```

Comment:
Could this file be reverted? As it looks like it's a formatting change

Reply:
I will revert this file.